aws/tf/modules/sra/variables.tf

```diff
@@ -82,6 +81,12 @@ variable "enable_cluster_boolean" {
   sensitive   = true
 }
 
+variable "enable_read_only_external_location" {
+  description = "Flag to enable read only external location"
```

Comment:
You mentioned this is a read only, optional bucket, so shouldn't we add a default value of false here that way you don't have to define this variable to use the module?

Reply:
I see its nullable so you can pass in null, but I think that's confusing.

Reply:
Right, I just mean that the best practice for writing a module being to provide default values for "optional" fields


Right now you have to include a line that sets it to false, which is how we have it structured in the example, but I think best practice is to set default variables here to indicate a true "optional" field


Also to indicate to any readers that that generally we would expect it to be false.